Modules/Filtering/Smoothing/test/itkMeanImageFilterTest.cxx

```diff
@@ -20,10 +20,14 @@
 #include "itkMeanImageFilter.h"
 #include "itkTextOutput.h"
 #include "itkTestingMacros.h"
+#include "itkVectorImage.h"
 
 int
 itkMeanImageFilterTest(int, char *[])
 {
+  using VectorImageType = itk::VectorImage<float, 3>;
+  itk::MeanImageFilter<VectorImageType, VectorImageType>::New();
```

Comment:
@blowekamp It would be really great if you could add some more elaborate tests for vector images than just doing `MeanImageFilter::New()` 😃 Just testing the filter for a few small vector images. If you like, there's a GTest for this filter as well.

Reply:
Working on it now...

Reply:
test added to the GTest file above.